sql2gee/collection.py

```diff
@@ -100,14 +102,21 @@ def _output(self):
         return _Output
 
     def _mapOutputIList(self, img):
-        if len(self._output['alias']['result']) > 0:
+        """
+        """
+        # @debt: inside the reducers we will need to use setOutputs
+        if len(self._output['alias']['result']) > 0 and type(img) is dict:
```

Comment:
is there any scenario where `type(img) is not dict` ? 
At least from the existing tests, it seems it's always a dictionary, so I'm wondering if this is redundant

Reply:
Is a pain in the ass the difference to type it in gee ee.Dictionary makes the operation happend on their server and not in ours

Reply:
ok, but does it ever happen that img is something other than a dict?